final-workflow.yml

```diff
@@ -76,16 +96,34 @@ deepPATH_bigwig: [ #the containing directory for bigwig files generated from the
 scaffold/bigwig
 ]
 #-------------------------------------------------------------------------------
-host: [ #host where Apollo2 lives
+#where apollo2 lives
+host_stage: [ 
 https://apollo-dev.nal.usda.gov
 ]
+host_production: [
+https://apollo-dev.nal.usda.gov
+]
+#authentication in apollo2
+login_apollo2_stage:  [
+user@ars.usda.gov,
+password
+]
+login_apollo2_production: [
+user@ars.usda.gov,
+password
+]
 #-------------------------------------------------------------------------------
-#authentication in apollo
+#Apollo server
 #username
-#password
-login_apollo2: [
-example@ars.usda.gov,
-ExamplePassword
-]
+Apollo_account: user@apollo-stage-node1.nal.usda.gov
+Gmod_account: user@gmod-stage-node1.nal.usda.gov
+Gmod_stage_account: user@gmod-dev.nal.usda.gov
 #-------------------------------------------------------------------------------
+#symlink PATH
+MAIN_PATH: /app/data/hsiukang.huang  
+LINK_DIR: apimel-(Apis_mellifera)/GCF_003254395.2
```

Comment:
@HsiuKangHuang could you change this so that LINK_DIR is automatically generated using tree[0] (e.g. apimel), scientific_name[0] and  scientific_name[1] (e.g. Apis_mellifera), and tree[1] (e.g. Amel_HAv3.1)? The last part is confusing, since in some cases we've used the RefSeq accession number but in other cases the assembly name for the directory. Let's stick with the assembly name (tree[1]) for this workflow.

Reply:
Okay!
I can change it.